### ISSUE TITLE
photon-controller-agent,envy and lightwave vibs installed sequentially

### ISSUE THREAD

**Issue by alancivita** [vmware/photon-controller#63]
Hi, I've noticed that the installation of the photon-controller-agent,envy and lightwave vibs happens sequentially for each cloud host; in case of a huge number of cloud hosts the installation could take ages...any plan to parallelise it?
Cheers
Alan

**Comment by roninby** [comment]
Yes, it took me about one and a half hour to deploy on 4 ESXi nodes. If auth is enabled, the vibs will be installed twice sequentially in each node.

**Comment by RedDevil69** [comment]
Hmmm strange for me such case takes around 15 min.
